content/applications/services/helpdesk.rst

```diff
@@ -32,7 +31,6 @@ team is assigned to, select it from the :guilabel:`Company` drop-down menu.
    use only.
 
    .. image:: helpdesk/team-description-webform.png
-      :align: center
       :alt: View of a Helpdesk team's website form displaying the team description.
 
 Visibility & Assignment
```

Comment:
I know we tend to de-capitalize text the after colons in bullet points, but because they're sentences, I would capitalize all of them. For example, in line 48, I would make it: 

:guilabel:`Invited internal users (private)`: Internal users can access the team and the tickets